resource/supported.go

```diff
@@ -595,6 +603,13 @@ func (a *AWS) s3Buckets() (interface{}, error) {
 	return output.Buckets, nil
 }
 
+func (a *AWS) trails() (interface{}, error) {
+	output, err := a.ListTrails(&cloudtrail.ListTrailsInput{})
```

Comment:
It's not obvious, but I think we should use the `DescribeTrails` function here instead.

Reply:
Done and done! Yeah, always imagined a `describe` would always end up with us having to manipulate or parse the output. Any reason behind the switch from your perspective?